dags/example_dynamodb.py

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from datetime import datetime
+
+import boto3
+from airflow.decorators import task
+from airflow.models.baseoperator import chain
+from airflow import DAG
+
+from airflow.providers.amazon.aws.sensors.dynamodb import DynamoDBValueSensor
+from tests.system.providers.amazon.aws.utils import ENV_ID_KEY, SystemTestContextBuilder
+
+DAG_ID = "example_dynamodbvaluesensor"
+sys_test_context_task = SystemTestContextBuilder().build()
+
+TABLE_ATTRIBUTES = [
+    {"AttributeName": "PK", "AttributeType": "S"},
+    {"AttributeName": "SK", "AttributeType": "S"},
+]
+TABLE_KEY_SCHEMA = [
+    {"AttributeName": "PK", "KeyType": "HASH"},
+    {"AttributeName": "SK", "KeyType": "RANGE"},
+]
+TABLE_THROUGHPUT = {"ReadCapacityUnits": 10, "WriteCapacityUnits": 10}
+
+
+@task
+def create_table(table_name: str):
+    ddb = boto3.resource("dynamodb")
+    table = ddb.create_table(
+        AttributeDefinitions=TABLE_ATTRIBUTES,
+        TableName=table_name,
+        KeySchema=TABLE_KEY_SCHEMA,
+        ProvisionedThroughput=TABLE_THROUGHPUT,
+    )
+    boto3.client("dynamodb").get_waiter("table_exists").wait()
+    table.put_item(Item={"PK": "Test", "SK": "2022-07-12T11:11:25-0400", "Value": "Testing"})
+
+
+@task
```

Comment:
```suggestion
@task(trigger_rule=TriggerRule.ALL_DONE)
```

This way, even if the test fails it will clean up the table.

Reply:
Done.